admin/installer/flocker-configuration-getter.sh

```diff
@@ -23,15 +23,25 @@ chmod --recursive a-rwx,u+rwX "${TMP_DIR}"
 mv "${TMP_DIR}" "${FLOCKER_CONFIG_DIRECTORY}"
 
 if test "${node_number}" -eq "0"; then
-    service flocker-control restart
+    # Enable flocker-control on node0 and disable the flocker agent services.
```

Comment:
There's a lot of copy-pasta here and weird abstractions.
It might be time to consider rewriting in Python, and having proper functions like "systemd_setup_service(name)" and "systemd_unsetup_service(name)" and then doing set-based service setup/unsetup so that the input can be more declarative.